### DISCUSSION TITLE
Adding PWA/SSB support natively

### DISCUSSION THREAD

**Discussion by droynvoss** [pulse-browser/browser#193] [Ideas]
Firefox,since the past couple of versions has not been supporting SSB (Single Site Browser), and requiring something like a Chromium PWA (Progressive Web App) needs workarounds or extensions such as [PWAsForFirefox](https://github.com/filips123/PWAsForFirefox).

It would be really useful if this support was added back. Would help with having applications (logged in) which don't have maintained software repositories for obscure Linux distributions.

**Comment by trickypr** [comment]
One of the other devs are looking into this, but we cannot promise anything at the moment. See pulse-browser/browser#39

**Comment by ghost** [comment]
+1
